compiler/damlc/tests/daml-test-files/InterfaceRetroRequiresError.daml

```diff
@@ -0,0 +1,33 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- @SINCE-LF-FEATURE DAML_INTERFACE
+-- @ERROR Template InterfaceRetroRequiresError:T is missing an implementation of interface InterfaceRetroRequiresError:A required by interface InterfaceRetroRequiresError:B
```

Comment:
Same remark as for compiler/damlc/tests/daml-test-files/InterfaceRetroMissingMethod.daml